pkg/loader/compose/compose.go

```diff
@@ -313,6 +476,125 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 		log.Warningf("Unsupported %s key - ignoring", keyName)
 	}
 
+	// Map the parsed struct to a struct we understand (kobject)
+	komposeObject, err := libComposeToKomposeMapping(composeObject)
+	if err != nil {
+		return kobject.KomposeObject{}, err
+	}
+
+	return komposeObject, nil
+}
+
+func dockerComposeToKomposeMapping(composeObject *types.Config) (kobject.KomposeObject, error) {
```

Comment:
looks a bit confusing IMO. method that I wrote looks pretty straight-forward to understand